### PR TITLE
eacl: Support new operators

### PR DESCRIPTION
**Pull request by cthulhu-rider**
- [x]  IR verification
- [x] SN processing (covered by https://github.com/nspcc-dev/neofs-sdk-go/pull/554)
- [x] CLI support

all ready, waiting for SDK merge

### CODE REVIEW

File: cmd/neofs-cli/modules/util/acl.go [hunk 1]
```diff
@@ -278,23 +289,52 @@ func parseEACLRecord(args []string) (*eacl.Record, error) {
...
-		op = eacl.MatchStringNotEqual
-	} else {
-		key = s[:i]
-		op = eacl.MatchStringEqual
+	if i == 0 {
+		return "", value, eacl.MatchStringEqual, nil
 	}
 
-	value = s[i+1:]
+	switch s[i-1] {
```
Comment (roman-khimov) [inline cmd/neofs-cli/modules/util/acl.go:317]: Some proper parser would be handy here, but luckily we don't have a lot of options now.

**Review by roman-khimov — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,12 @@ Changelog for NeoFS Node
 ## [Unreleased]
 
 ### Added
+- SN eACL processing of NULL and numeric operators (#2742)
```
Comment (carpawell) [inline CHANGELOG.md:7]: the same commit as in https://github.com/nspcc-dev/neofs-node/pull/2733 but with a different code?

File: pkg/innerring/processors/container/process_eacl.go [hunk 3]
```diff
@@ -99,12 +100,27 @@ func (cp *Processor) approveSetEACL(e container.SetEACL) {
...
 func validateEACL(t *eacl.Table) error {
+	var b big.Int
 	for _, record := range t.Records() {
 		for _, target := range record.Targets() {
 			if target.Role() == eacl.RoleSystem {
 				return errors.New("it is prohibited to modify system access")
 			}
 		}
+		for _, f := range record.Filters() {
+			//nolint:exhaustive
```
Comment (carpawell) [inline pkg/innerring/processors/container/process_eacl.go:111]: why not two `if`s or `default`?

File: CHANGELOG.md [hunk 4]
```diff
@@ -30,6 +32,9 @@ raw binaries. All binaries have OS in their names as well now, following
 regular naming used throughout NSPCC, so instead of neofs-cli-amd64 you get
 neofs-cli-linux-amd64 now.
 
+CLI command `acl extended create` changed and extended input format for filters.
+For example, `attr>=100` or `attr=` are now processed differently. See `-h` for details.
```
Comment (carpawell) [inline CHANGELOG.md:36]: "See `-h`"? see help? use -h flag?

File: cmd/neofs-cli/modules/util/acl.go [hunk 5]
```diff
@@ -278,23 +289,52 @@ func parseEACLRecord(args []string) (*eacl.Record, error) {
 func parseKVWithOp(s string) (string, string, eacl.Match, error) {
 	i := strings.Index(s, "=")
 	if i < 0 {
+		if i = strings.Index(s, "<"); i >= 0 {
+			if !validateDecimal(s[i+1:]) {
+				return "", "", 0, fmt.Errorf("invalid base-10 integer value %q for attribute %q", s[i+1:], s[:i])
+			}
+			return s[:i], s[i+1:], eacl.MatchNumLT, nil
```
Comment (carpawell) [inline cmd/neofs-cli/modules/util/acl.go:296]: what about `<=`?

File: CHANGELOG.md (hunk 2 above)
Comment (cthulhu-rider) [reply CHANGELOG.md:7]: > the same commit

what do u mean?

File: pkg/innerring/processors/container/process_eacl.go (hunk 3 above)
Comment (cthulhu-rider) [reply pkg/innerring/processors/container/process_eacl.go:111]: not two ifs cuz more than 2 values, not default cuz it'd be empty. IMO that's not the case where linter suggests smth meaningful

File: CHANGELOG.md (hunk 4 above)
Comment (cthulhu-rider) [reply CHANGELOG.md:36]: see help text

File: cmd/neofs-cli/modules/util/acl.go (hunk 5 above)
Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/util/acl.go:296]: `i < 0` is false for it

File: pkg/innerring/processors/container/process_eacl.go (hunk 3 above)
Comment (carpawell) [reply pkg/innerring/processors/container/process_eacl.go:111]: > not two ifs cuz more than 2 values

every `case` has `return`, still can be with `if`s

> not default cuz it'd be empty

more regular expression for a regular go reader (even more if a comment is just `nolint` without some real comment). both are one line long (`deafult` always shorter)

File: CHANGELOG.md (hunk 4 above)
Comment (carpawell) [reply CHANGELOG.md:45]: ?

File: cmd/neofs-cli/modules/util/acl.go (hunk 5 above)
Comment (carpawell) [reply cmd/neofs-cli/modules/util/acl.go:296]: did not see the if below

Comment (carpawell) [reply cmd/neofs-cli/modules/util/acl.go:296]: did not see the if below

File: pkg/innerring/processors/container/process_eacl.go (hunk 3 above)
Comment (carpawell) [reply pkg/innerring/processors/container/process_eacl.go:111]: well, can be merged, but i do not understand it (not disagree just don't understand why it is easier to do than one of my suggestion)

File: CHANGELOG.md (hunk 4 above)
Comment (cthulhu-rider) [reply CHANGELOG.md:45]: `-h` prints help text that u can see

Comment (carpawell) [reply CHANGELOG.md:45]: i mean i cant remember i see somewhere "see -h". it is usually "see help" or "use -h flag for details"

**Review by carpawell — APPROVED** [review]

File: pkg/innerring/processors/container/process_eacl.go (hunk 3 above)
Comment (cthulhu-rider) [reply pkg/innerring/processors/container/process_eacl.go:111]: > more regular expression for a regular go reader

empty `default` is useless and always skip, that's what i see as a regular Go reader

File: CHANGELOG.md (hunk 4 above)
Comment (cthulhu-rider) [reply CHANGELOG.md:45]: i bet anybody will understand what is this about

**Pull request merged**